core/src/main/scala/org/bitcoins/core/protocol/ln/routing/LnRoute.scala

```diff
@@ -25,9 +25,6 @@ case class LnRoute(
     cltvExpiryDelta: Short)
     extends NetworkElement {
 
-  require(pubkey.isCompressed,
```

Comment:
Is it safe this invariant is gone? Can we use non compressed keys in routing?

Reply:
`ECPublicKey` no longer represents a specific serialization, calling `_.bytes` will always return a compressed public key.